integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystemOpts.java

```diff
@@ -0,0 +1,155 @@
+        conf.getString(PropertyKey.FUSE_MOUNT_ALLUXIO_PATH),
+        conf.getString(PropertyKey.FUSE_MOUNT_POINT),
+        fuseOptions,
+        conf.getBoolean(PropertyKey.FUSE_DEBUG_ENABLED)
+    );
+  }
+
+  /**
+   * Constructs an AlluxioFuseFileSystemOpts with Alluxio cluster configuration and command line.
+   * Command line input has higher precedence if a property is set both in config and command.
+   *
+   * @param conf     Alluxio cluster configuration
+   * @param fuseCliOpts     Alluxio fuse command line input
+   * @return AlluxioFuseFileSystemOpts
+   */
+  public static AlluxioFuseFileSystemOpts create(
+      AlluxioConfiguration conf, AlluxioFuseCliOpts fuseCliOpts) {
+    Optional<String> alluxioPathFromCli = fuseCliOpts.getMountAlluxioPath();
+    String alluxioPath = alluxioPathFromCli.orElseGet(
+        () -> conf.getString(PropertyKey.FUSE_MOUNT_ALLUXIO_PATH));
+    Optional<String> mountPointFromCli = fuseCliOpts.getMountPoint();
+    String mountPoint = mountPointFromCli.orElseGet(
+        () -> conf.getString(PropertyKey.FUSE_MOUNT_POINT));
+    Optional<List<String>> libfuseOptionsFromCli = fuseCliOpts.getFuseOptions();
+    List<String> libfuseOptions;
+    if (libfuseOptionsFromCli.isPresent()) {
+      libfuseOptions = libfuseOptionsFromCli.get();
+    } else {
+      if (conf.isSet(PropertyKey.FUSE_MOUNT_OPTIONS)) {
+        libfuseOptions = conf.getList(PropertyKey.FUSE_MOUNT_OPTIONS);
+      } else {
+        libfuseOptions = ImmutableList.of();
+      }
+    }
+    libfuseOptions = optimizeAndFormatFuseOptions(libfuseOptions);
+    boolean isDebug = conf.getBoolean(PropertyKey.FUSE_DEBUG_ENABLED);
+
+    return new AlluxioFuseFileSystemOpts(alluxioPath, mountPoint, libfuseOptions, isDebug);
+  }
+
+  /**
+   * Constructs an AlluxioFuseFileSystemOpts solely for testing purpose.
+   *
+   * @param  alluxioPath
+   * @param  mountPoint
+   * @param  fuseOptions
+   * @param  isDebug
+   * @return AlluxioFuseFileSystemOpts
+   */
+  @VisibleForTesting
+  public static AlluxioFuseFileSystemOpts create(
+      String alluxioPath, String mountPoint, List<String> fuseOptions, boolean isDebug) {
+    return new AlluxioFuseFileSystemOpts(alluxioPath, mountPoint, fuseOptions, isDebug);
+  }
+
+  private AlluxioFuseFileSystemOpts(
+      String alluxioPath, String mountPoint, List<String> fuseOptions, boolean isDebug) {
+    mAlluxioPath = alluxioPath;
+    mMountPoint = mountPoint;
+    mLibfuseOptions = fuseOptions;
+    mIsDebug = isDebug;
+  }
+
+  /**
+   * @return the alluxio path
+   */
+  public String getAlluxioPath() {
+    return mAlluxioPath;
+  }
+
+  /**
+   * @return the mount point
+   */
+  public String getMountPoint() {
+    return mMountPoint;
+  }
+
+  /**
+   * @return options for Libfuse
+   */
+  public List<String> getFuseOptions() {
+    return mLibfuseOptions;
+  }
+
+  /**
+   * @return if using debug-level logging for AlluxiJniFuseFileSystem and Libfuse
```

Comment:
`integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystemOpts.java` make it sound like it's for AlluxioFuseFileSystem lol

Reply:
It's for both, so I renamed the AlluxioFuseFileSystem to AlluxioJnrFuseFileSystem